index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - setup and mobile first</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;600&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="toolbar">
+            <div class="container-1">
+                <div class="clock"><p><strong>9:41</strong></p></div>
+                <div class="space2"></div>
+                <div class="conn"><img class="conn-im" alt="Connetion" src="images/connection.png"></div>
+                <div class="wire"><img class="wire-im" alt="Wireless" src="images/wireless.png"></div>
+                <div class="batt"><img class="batt-im" alt="Battery" src="images/battery.png"></div>
+            </div>
+            <div class="container-2">
+                <div class="logo"><img class="mylogo-im" alt="MyLogo" src="images/mylogo.png"></div>
+                <div class="space1"></div>
+                <div class="menu"><img class="menu-im" alt="Menu" src="images/menu.png"></div>
+            </div>
+        </div>
+    </header>
+    <section class="headline">
+        <h1>I'm Juan Carlos. Glad to see you!</h1>
+        <p class="desc">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <p class="connect"><strong>LET'S CONNECT</strong></p>
+        <div class="container-3">
+            <a class="twitter" href="www.twitter.com" target="_blank"><img alt="twitter" src="images/twitter.png"></a>
+            <a class="linkedin" href="www.linkedin.com" target="_blank"><img class="li" alt="linkedin" src="images/linkedin.png"></a>
+            <a class="m" href="www.google.com" target="_blank"><img alt="logo-m" src="images/m.png"></a>
+            <a class="github" href="www.github.com" target="_blank"><img alt="github" src="images/github.png"></a>
+            <a class="sign" href="www.google.com" target="_blank"><img alt="logo-sign" src="images/sign.png"></a>
```

Comment:
- [ ] For this part, kindly make sure the list of icons uses a `ul` tag to group, and `li` per icon; each icon has a link (`a` tag).

- Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.